ui/tutorial-renderer/src/main/res/drawable/img_baked_in_others.xml

```diff
@@ -0,0 +1,107 @@
+    <aapt:attr name="android:fillColor">
+      <gradient 
+          android:startY="10.5173"
+          android:startX="44.7002"
+          android:endY="47.3198"
+          android:endX="32.4143"
+          android:type="linear">
+        <item android:offset="0" android:color="#BF3BA4DB"/>
+        <item android:offset="0.84375" android:color="#FFFFFFFF"/>
+      </gradient>
+    </aapt:attr>
+  </path>
+  <path
+      android:pathData="M64.75,41.546C64.75,34.801 69.689,33.452 69.689,19.288C69.689,19.288 72.57,21.986 72.57,33.452C72.57,40.197 68.043,41.546 68.043,55.036C68.043,55.036 64.75,52.338 64.75,41.546Z">
+    <aapt:attr name="android:fillColor">
+      <gradient 
+          android:startY="20.4286"
+          android:startX="72.619"
+          android:endY="53.0374"
+          android:endX="61.5604"
+          android:type="linear">
+        <item android:offset="0" android:color="#BF3BA4DB"/>
+        <item android:offset="0.84375" android:color="#FFFFFFFF"/>
+      </gradient>
+    </aapt:attr>
+  </path>
+  <path
+      android:pathData="M50.227,32.988C50.227,23.082 56.576,21.101 56.576,0.297C56.576,0.297 60.281,4.259 60.281,21.101C60.281,31.007 54.46,32.988 54.46,52.801C54.46,52.801 50.227,48.839 50.227,32.988Z">
+    <aapt:attr name="android:fillColor">
+      <gradient 
+          android:startY="1.97241"
+          android:startX="60.3439"
+          android:endY="48.4057"
+          android:endX="42.3551"
+          android:type="linear">
+        <item android:offset="0" android:color="#BF3BA4DB"/>
+        <item android:offset="0.84375" android:color="#FFFFFFFF"/>
+      </gradient>
+    </aapt:attr>
+  </path>
+  <path
+      android:pathData="M109.434,72.256C109.434,64.668 115.078,63.151 115.078,47.216C115.078,47.216 118.371,50.251 118.371,63.151C118.371,70.739 113.197,72.256 113.197,87.432C113.197,87.432 109.434,84.397 109.434,72.256Z">
+    <aapt:attr name="android:fillColor">
+      <gradient 
+          android:startY="48.4992"
+          android:startX="118.427"
+          android:endY="85.3017"
+          android:endX="106.141"
+          android:type="linear">
+        <item android:offset="0" android:color="#BF3BA4DB"/>
+        <item android:offset="0.84375" android:color="#FFFFFFFF"/>
+      </gradient>
+    </aapt:attr>
+  </path>
+  <path
+      android:pathData="M100.5,68.357C100.5,61.612 105.439,60.263 105.439,46.099C105.439,46.099 108.32,48.797 108.32,60.263C108.32,67.008 103.793,68.357 103.793,81.847C103.793,81.847 100.5,79.149 100.5,68.357Z">
+    <aapt:attr name="android:fillColor">
+      <gradient 
+          android:startY="47.2397"
+          android:startX="108.369"
+          android:endY="79.8484"
+          android:endX="97.3104"
+          android:type="linear">
+        <item android:offset="0" android:color="#BF3BA4DB"/>
+        <item android:offset="0.84375" android:color="#FFFFFFFF"/>
+      </gradient>
+    </aapt:attr>
+  </path>
+</vector>
```

Comment:
this is a massive vector image, is it something that would make more sense as a jpg or png?

Reply:
<img width="640" alt="godtools____Documents_GitHub_godtools-android__-_____ui_tutorial-renderer_src_main_res_drawable_img_baked_in_others_xml__ui-tutorial-renderer_" src="https://user-images.githubusercontent.com/970048/70356491-89618800-1842-11ea-940f-d0e87cc2987a.png">

This is the image let me know what you would like me to do?